### PR TITLE
Lintfix20221003

### PR DESCRIPTION
**Pull request by KVVat**
Fix build errors of andoird security samples.

### CODE REVIEW

File: DirectBoot/Application/src/main/java/com/example/android/directboot/alarms/AlarmAdapter.java [hunk 1]
```diff
@@ -67,8 +69,8 @@ public AlarmViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     }
 
     @Override
-    public void onBindViewHolder(AlarmViewHolder holder, final int position) {
-        Alarm alarm = mAlarmList.get(position);
+    public void onBindViewHolder(AlarmViewHolder holder,@SuppressLint("RecyclerView")  final int position) {
+        Alarm alarm = mAlarmList.get(holder.getAdapterPosition());
```
Comment (jdkoren) [inline DirectBoot/Application/src/main/java/com/example/android/directboot/alarms/AlarmAdapter.java:73]: Is there a reason not to use `position` here? The `position` argument should determine which item in the list to use.

File: DirectBoot/Application/src/main/java/com/example/android/directboot/alarms/AlarmAdapter.java [hunk 2]
```diff
@@ -67,8 +69,8 @@ public AlarmViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     }
 
     @Override
-    public void onBindViewHolder(AlarmViewHolder holder, final int position) {
-        Alarm alarm = mAlarmList.get(position);
+    public void onBindViewHolder(AlarmViewHolder holder,@SuppressLint("RecyclerView")  final int position) {
```
Comment (jdkoren) [inline DirectBoot/Application/src/main/java/com/example/android/directboot/alarms/AlarmAdapter.java:72]: What is the warning being suppressed? (what does it complain about if you remove the `@SuppressLint` annotation?)

File: DirectBoot/Application/src/main/res/layout/fragment_alarm_scheduler.xml [hunk 3]
```diff
@@ -39,15 +39,15 @@
...
         android:id="@+id/recycler_view_alarms"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:layout_below="@id/text_intro_message"
         />
 
-    <android.support.design.widget.FloatingActionButton
-        android:id="@+id/fab_add_alarm"
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+    android:id="@+id/fab_add_alarm"
```
Comment (jdkoren) [inline DirectBoot/Application/src/main/res/layout/fragment_alarm_scheduler.xml:50]: nit: please fix indentation

File: DirectBoot/Application/src/main/java/com/example/android/directboot/alarms/AlarmAdapter.java (hunk 1 above)
Comment (KVVat) [reply DirectBoot/Application/src/main/java/com/example/android/directboot/alarms/AlarmAdapter.java:73]: Hi jd
https://stackoverflow.com/questions/34942840/lint-error-do-not-treat-position-as-fixed-only-use-immediately/34943270#34943270

Comment (KVVat) [reply DirectBoot/Application/src/main/java/com/example/android/directboot/alarms/AlarmAdapter.java:73]: Yes. you are right, in this context.  The change is intended to suppress a lint error to pass CI checks, though there is no problem with behaviour. 

[StackOverFlow](https://stackoverflow.com/questions/34942840/lint-error-do-not-treat-position-as-fixed-only-use-immediately/34943270#34943270
)
[Official Documentation](https://developer.android.com/reference/android/support/v7/widget/RecyclerView.Adapter.html#onbindviewholder_1)

Thanks

File: DirectBoot/Application/src/main/res/layout/fragment_alarm_scheduler.xml (hunk 3 above)
Comment (KVVat) [reply DirectBoot/Application/src/main/res/layout/fragment_alarm_scheduler.xml:50]: Thanks! Will update it.

File: DirectBoot/Application/src/main/java/com/example/android/directboot/alarms/AlarmAdapter.java (hunk 1 above)
Comment (KVVat) [reply DirectBoot/Application/src/main/java/com/example/android/directboot/alarms/AlarmAdapter.java]: sorry it's my misunderstanding... I revert this change.

File: Fido/gradle.properties [hunk 4]
```diff
@@ -10,6 +10,8 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx10248m -XX:MaxPermSize=256m
+android.enableJetifier=true
```
Comment (jdkoren) [inline Fido/gradle.properties:13]: Line 15 should be directly under the comment above. Let's move lines 13 and 14 to the bottom of the file.

File: DirectBoot/gradle.properties [hunk 5]
```diff
@@ -1,20 +1,2 @@
-
```
Comment (jdkoren) [inline DirectBoot/gradle.properties:1]: Should we keep these comments?

File: Fido/gradle.properties (hunk 4 above)
Comment (KVVat) [reply Fido/gradle.properties:14]: Thanks. Changed the order of lines.

File: DirectBoot/gradle.properties (hunk 5 above)
Comment (KVVat) [reply DirectBoot/gradle.properties]: Thanks. It dissppeared because of the AndroidX migration tool.  Fixed.

**Review by jdkoren — APPROVED** [review]

**Pull request merged**